services/web/README.md

```diff
@@ -0,0 +1,8 @@
+# Web service
+
+The web service embeds and serves the static files of the [Infinite Scale Web Client](https://github.com/owncloud/web).  
+It also provides a minimal API for branding functionality like changing the logo.
+
+# Custom compiled Web assets
```

Comment:
```suggestion
# Web Service

The web service embeds and serves the static files of the [Infinite Scale Web Client](https://github.com/owncloud/web).  
It also provides a minimal API for branding functionality like changing the logo.

## Custom Compiled Web Assets
```
* language phrasing question:
`embeds and serves the static files of the`
I am not 100% sure, but I think it should be: 
`embeds and serves the static files for the` 
* what happens if the service is down?
does any new client that connects to ocis break down?